tests/sentry/models/test_groupsubscription.py

```diff
@@ -24,6 +24,67 @@ def test_simple(self):
         # should not error
         GroupSubscription.objects.subscribe(group=group, user=user)
 
+    def test_bulk(self):
+        group = self.create_group()
+
+        user_ids = []
+        for i in range(20):
+            user = self.create_user()
+            user_ids.append(user.id)
+
+        GroupSubscription.objects.bulk_subscribe(group=group, user_ids=user_ids)
+
+        assert len(GroupSubscription.objects.filter(
+            group=group,
+        )) is 20
```

Comment:
I'm not 100% confident on this, but I think it's possible for this to not evaluate as you expect. I'd switch these comparisons to `==`. I don't know if it's guaranteed that integers will always occupy the same exact memory address.

`is` compares an exact piece of memory. Objects only pass `is` check if they are, in fact, the exact same object. Not just the same value.

Reply:
damn. cool